examples/app-with-scenes/package.json

```diff
@@ -58,11 +58,11 @@
   },
   "dependencies": {
     "@emotion/css": "^11.1.3",
-    "@grafana/data": "9.5.1",
-    "@grafana/runtime": "9.5.1",
-    "@grafana/scenes": "0.11.0",
-    "@grafana/schema": "9.5.1",
-    "@grafana/ui": "9.5.1",
+    "@grafana/data": "10.0.0",
+    "@grafana/runtime": "10.0.0",
+    "@grafana/scenes": "0.16.0",
+    "@grafana/schema": "canary",
```

Comment:
@jackw - I know this ain't the best, but until 10.0.1 is out we need to rely on canary :(

Reply:
No worries @dprokop we'll update it when the time comes. Can you please run `npm install` in the scenes app directory to update the package-lock.json file so CI can do it's thing? 🤖